app/containers/ItunesContainer/index.js

```diff
@@ -116,7 +117,8 @@ export function ItunesContainer({
     currentAudio.play();
   };
 
-  const pauseSong = () => {
+  const pauseSong = (event) => {
+    event.preventDefault();
```

Comment:
Ideally, I would move this to the other component. The prevent default has nothing to do with any of the UI of this component, but rather a click event of the `TrackCard` component.